automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -41,24 +41,6 @@ class MethodConfigSpec extends FreeSpec with WebBrowserSpec with CleanUp with Wo
     }
   }
 
-  "launch modal with no default entities" ignore withWebDriver { implicit driver =>
```

Comment:
you can no longer launch a workflow with no default entity. other test cases exist to test the other error scenarios like trying to launch a workflow with the wrong entity type selected.

Reply:
I believe this test case was to test opening the modal when there are no entities. The expected result is that there's some helpful notification that the user has no data. (There has been a bug with this before.)
Another question we should ask is whether this is a test worthy of upkeep and whether it would block a release if it were broken.